module/dkg/controller.go

```diff
@@ -11,6 +14,34 @@ import (
 	"github.com/onflow/flow-go/module"
 )
 
+const (
+
+	// DefaultBaseStartDelay is the default base delay to use when introducing
+	// random delay to the DKG start process. See preStartDelay for details.
+	DefaultBaseStartDelay = 500 * time.Microsecond
+
+	// DefaultBaseHandleBroadcastDelay is the default base to use when
+	// introducing random delay to processing EACH DKG broadcast message. See
+	// preHandleBroadcastDelay for details.
+	//
+	// For a 150-node DKG, we observe a cost of ~2.5s per message to process
+	// broadcast messages during phase 1, for a total of ~6m of total CPU time.
+	// We would like to target spreading this cost over a 20 minute period in the
+	// average case.
+	//
+	// 500µs results in per-broadcast delays of max=11.25s, ave=5.625s.
```

Comment:
What is the 500µs? Or are you talking about 500ms that is used as the DefaultBaseHandleBroadcastDelay
```suggestion
	// 500ms results in per-broadcast delays of max=11.25s, ave=5.625s.
```

Reply:
`DefaultBaseHandleBroadcastDelay` is 500 microseconds, not 500 milliseconds ([code](https://github.com/onflow/flow-go/pull/1303/files#diff-0e1050f66811bb0fd32918920021ad123932be01f04472386ea9ba5df2a6056aR36))